app/code/Magento/Theme/Block/Html/Topmenu.php

```diff
@@ -394,4 +394,17 @@ public function getMenu()
         }
         return $this->_menu;
     }
+    
+    /**
+     * Get cache key informative items
+     *
+     * @return array
+     * @since 100.1.0
```

Comment:
This is not an introduction of the method, but just an override, so `@since` annotation is not necessary